wagtail/images/models.py

```diff
@@ -1408,6 +1408,62 @@ def background_position_style(self):
         else:
             return "background-position: 50% 50%;"
```

Comment:
We can refactor this (and the above lines) to use the newly added `background_position_x` and `background_position_y`, e.g.

```py
horz = self.background_position_x
vert = self.background_position_y
return f"background-position: {horz} {vert};
```